core/src/test/java/org/apache/calcite/test/LintTest.java

```diff
@@ -393,6 +395,27 @@ private static void checkMessage(String subject, String body,
     if (subject2.matches("[a-z].*")) {
       consumer.accept("Message must start with upper-case letter");
     }
+
+    // Check for keywords that should be capitalized
```

Comment:
Consider moving keywordMap, and its population, outside the loop.

Reply:
done